pulsar-common/src/main/java/org/apache/pulsar/common/util/collections/OpenLongPairRangeSet.java

```diff
@@ -54,15 +56,15 @@ public class ConcurrentOpenLongPairRangeSet<T extends Comparable<T>> implements
     private volatile boolean updatedAfterCachedForSize = true;
     private volatile boolean updatedAfterCachedForToString = true;
 
-    public ConcurrentOpenLongPairRangeSet(LongPairConsumer<T> consumer) {
+    public OpenLongPairRangeSet(LongPairConsumer<T> consumer) {
         this(1024, true, consumer);
```

Comment:
Just wondering if thread safe option should be removed or default to false?

Reply:
Maybe in the followup PR(RoaringBitSet)?